rimage/cmd/stream_camera/main.go

```diff
@@ -87,7 +87,7 @@ func mainWithArgs(ctx context.Context, args []string, logger golog.Logger) error
 }
 
 func viewCamera(ctx context.Context, attrs videosource.WebcamAttrs, port int, debug bool, logger golog.Logger) error {
-	webcam, err := videosource.NewWebcamSource(ctx, &attrs, logger)
+	webcam, err := videosource.NewWebcamSource(ctx, &attrs, "viam_webrtc_camera", logger)
```

Comment:
Thank you for looking into that. Just to be sure to understand, your suggestion is to replace `"viam_webrtc_camera" `by `camera.Named("model").String() `? If not, what `model` is referring to? 

Concerning using `attrs.Path` to identify webcams in the logs, I thought it might be confusing if two webcams have the same path (if one of them is a remote for instance).

Reply:
At the top of webcam.go `model` is just a const
```
const model = "webcam"
```
So you could use the `camera.Named(model)` line in that file on line 269 rather than passing a string from `videosource.NewWebcamSource` down.

Reply:
I guess we wanted to have the name of the failing camera in the error message. However the camera name will be displayed in the logs in the `resource` field.

So I can remove `name` from the arguments of `videosource.NewWebcamSource`  and have the error message like:

`error building component
resource: rdk:component:camera/**name_of_the_webcam**
model: webcam
error: could not find video source for camera: requested width and height (1000x1000) are not available (closest driver found by gostream supports resolution 640x480)`

instead of:

`error building component
resource: rdk:component:camera/**name_of_the_webcam**
model: webcam
error: could not find video source for camera: requested width and height (1000x1000) are not available for webcam **name_of_the_webcam** (closest driver found by gostream supports resolution 640x480)`

Sounds good?

Reply:
Yeah, I think that makes sense. I do wonder if you could use that same resource field too but I'm not sure how that's grabbed. Either is fine by me. Was just worried that a user might see "viam_webrtc_camera" and not understand what that means and how it relates to the other names we give cameras in our logs. Sometimes we call it by its path, other times by its resource name.